sql_generators/active_users_versioned/templates/mobile_query2.sql

```diff
@@ -0,0 +1,495 @@
+        ELSE languages.language_name
+      END AS language_name,
+    FROM
+      todays_metrics
+    LEFT JOIN
+      `mozdata.static.csa_gblmkt_languages` AS languages
+    ON
+      todays_metrics.locale = languages.code
+  )
+  SELECT
+    segment,
+    attribution_medium,
+    attribution_source,
+    attributed,
+    city,
+    country,
+    distribution_id,
+    first_seen_year,
+    is_default_browser,
+    channel,
+    os,
+    os_version,
+    os_version_major,
+    os_version_minor,
+    submission_date,
+    language_name,
+    adjust_network,
+    install_source,
+    app_name,
+    app_version,
+    device_manufacturer,
+    CURRENT_TIMESTAMP AS last_updated_timestamp,
+    COUNT(DISTINCT IF(days_since_seen = 0, client_id, NULL)) AS dau,
+    COUNT(DISTINCT IF(days_since_seen < 7, client_id, NULL)) AS wau,
+    COUNT(DISTINCT client_id) AS mau,
+    SUM(uri_count) AS uri_count,
+    SUM(active_hours_sum) AS active_hours
+  FROM
+    todays_metrics_enriched
+  GROUP BY
+    app_version,
+    attribution_medium,
+    attribution_source,
+    attributed,
+    city,
+    country,
+    distribution_id,
+    first_seen_year,
+    is_default_browser,
+    language_name,
+    app_name,
+    channel,
+    os,
+    os_version,
+    os_version_major,
+    os_version_minor,
+    submission_date,
+    segment,
+    adjust_network,
+    install_source,
+    device_manufacturer,
+    last_updated_timestamp
+)
+SELECT
+  segment,
+  attribution_medium,
+  attribution_source,
+  attributed,
+  city,
+  country,
+  distribution_id,
+  first_seen_year,
+  is_default_browser,
+  channel,
+  os,
+  os_version,
+  os_version_major,
+  os_version_minor,
+  submission_date,
+  language_name,
+  adjust_network,
+  install_source,
+  app_name,
+  app_version,
+  ARRAY_AGG(
+    STRUCT(device_manufacturer, dau, wau, mau, uri_count, active_hours)
+  ) AS device_manufacturer,
+  CURRENT_TIMESTAMP AS last_updated_timestamp,
+  SUM(dau) AS dau,
+  SUM(wau) AS wau,
+  SUM(mau) AS mau,
+  SUM(uri_count) AS uri_count,
+  SUM(active_hours) AS active_hours
+FROM
+  _current_base
+GROUP BY
+  segment,
+  attribution_medium,
+  attribution_source,
+  attributed,
+  city,
+  country,
+  distribution_id,
+  first_seen_year,
+  is_default_browser,
+  channel,
+  os,
+  os_version,
+  os_version_major,
+  os_version_minor,
+  submission_date,
+  language_name,
+  adjust_network,
+  install_source,
+  app_name,
+  app_version;
+
+CREATE TEMP TABLE
+  _previous
+AS
+SELECT
+  * EXCEPT (
+    new_profiles,
+    ad_clicks,
+    organic_search_count,
+    search_count,
+    search_with_ads,
+    app_version_major,
+    app_version_minor,
+    app_version_patch_revision,
+    app_version_is_major_release,
+    os_grouped
+  ),
+  CAST(NULL AS STRING) AS device_manufacturer,
+  DATE_SUB(CURRENT_TIMESTAMP, INTERVAL 1 DAY) AS last_updated_timestamp
+FROM
+  `moz-fx-data-shared-prod.focus_ios.active_users_aggregates`
+WHERE
+  submission_date
+  BETWEEN '2023-01-01'
```

Comment:
Testing period. In the final query this will be `submission_date >= '2021-01-01'`